e2e/tests/stepDefinitions/manage.ts

```diff
@@ -6,14 +6,36 @@ import bookingFactory from '../../../server/testutils/factories/booking'
 import personFactory from '../../../server/testutils/factories/person'
 import keyWorkerFactory from '../../../server/testutils/factories/keyWorker'
 
-const keyWorker = keyWorkerFactory.build({ name: Cypress.env('keyworker_name') })
-const person = personFactory.build({ name: Cypress.env('offender_name'), crn: Cypress.env('offender_crn') })
+const keyWorkerName = Cypress.env('keyworker_name')
+const offenderName = Cypress.env('offender_name')
+const offenderCrn = Cypress.env('offender_crn')
+
+// eslint-disable-next-line no-unused-expressions
+expect(keyWorkerName, 'keyWorkerName was set').to.be.a('string').and.not.be.empty
```

Comment:
This might be more readable if we had a function to throw an error like:

```typescript
const throwMissingError(field: string) {
  throw new Error(`Missing Cypress env variable for '${field}'`)
}
```

And then we could assign and throw if nil in one line like so:

```typescript
const keyWorkerName = Cypress.env('keyworker_name') || throwMissingError('keyworker_name')
```

Reply:
(Note: we can't do `const keyWorkerName = Cypress.env('keyworker_name') || throw new Error(`Missing Cypress env variable for 'keyworker_name'`)` because `throw` is a statement, and can't (yet) be used as an operand to an operator.